index.ts

```diff
@@ -195,6 +200,39 @@ function getAndDisplayWeather(
   }
 }
 
+function getAndDisplayIndividualStockData(
+  message: Discord.Message,
+  args: string[]
+): Promise<Discord.Message | Discord.Message[]> {
+  if (!args.length) {
+    return message.channel.send(
+      `\`\`\`You didn't provide enough arguments, requires: <!stocks STOCKSYMBOL>, e.g. <!stocks TWTR>\`\`\``
+    );
+  } else {
+    const [symbol] = args;
+
+    stocksAPI.getStockData(symbol).then(stockResponse => {
+      const discordResponseStockData: object = {
```

Comment:
Notice how you are calling `stockResponse.data[0]` a lot? Pull that out into a variable, to save some keystrokes.

I.e. 

`const data = stockResponse.data[0]`